exe/floe

```diff
@@ -7,15 +7,34 @@ require "optimist"
 opts = Optimist.options do
   version "v#{Floe::VERSION}\n"
   opt :workflow, "Path to your workflow json", :type => :string, :required => true
-  opt :inputs, "JSON payload to input to the workflow", :type => :string, :default => '{}'
-  opt :credentials, "JSON payload with credentials", :type => :string, :default => '{}'
+  opt :inputs, "JSON payload to input to the workflow", :type => :string, :default => "{}"
+  opt :credentials, "JSON payload with credentials", :type => :string, :default => "{}"
+  opt :runner, "Type of runner for docker images", :type => :string, :default => "docker"
+  opt :runner_option, "Options to pass to the runner", :type => :string, :multi => true
 end
 
 require "logger"
 Floe.logger = Logger.new(STDOUT)
 
 workflow = Floe::Workflow.load(opts[:workflow], opts[:inputs], opts[:credentials])
 
+if opts.key?(:runner)
```

Comment:
Since you've specified a default then I don't thing there's ever a way for this not to be true.